.github/workflows/release.yml

```diff
@@ -110,6 +110,7 @@ jobs:
           upgrade-commands: ${{ inputs.upgrade-commands }}
           allow-major: false
           package-json-path: ${{ inputs.package-json-path }}
+          urgent: ${{ github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'pco-release-urgent') }}
```

Comment:
I might suggest adding some sort of indicator to the comment that gets added to the origin repo that the release was deployed as `urgent` so there's more understanding what the github action did

Reply:
I think as a first step, I'm going to merge this work in since it works as is.  I created an [Asana task](https://app.asana.com/1/1202585680506197/project/1203135756039983/task/1210676896005873?focus=true) to address this.